quill-core/src/main/scala/io/getquill/sources/QueryMacro.scala

```diff
@@ -16,17 +16,18 @@ trait QueryMacro extends SelectFlattening with SelectResultExtraction {
     quotedTree:     Tree,
     ast:            Ast,
     inPlaceParams:  collection.Map[Ident, (Type, Tree)],
-    functionParams: List[(Ident, Type)]
+    functionParams: List[(Ident, Type)],
+    returnList:     Boolean                             = true
```

Comment:
Let's avoid the default value here please, it's better if the callers define this explicitly.